frontend/server/src/main/java/com/amazonaws/ml/mms/http/messages/RegisterModelRequest.java

```diff
@@ -60,6 +63,7 @@ public RegisterModelRequest(QueryStringDecoder decoder) {
         synchronous = Boolean.parseBoolean(NettyUtils.getParameter(decoder, "synchronous", "true"));
         responseTimeout = NettyUtils.getIntParameter(decoder, "response_timeout", -1);
         modelUrl = NettyUtils.getParameter(decoder, "url", null);
+        preloadModel = NettyUtils.getParameter(decoder, "preload_model", "false");
```

Comment:
This adds new parameter to the register model API. If this is fine, we would need to update the management open API doc](https://github.com/awslabs/mxnet-model-server/blob/master/frontend/server/src/test/resources/management_open_api.json) and also the [OpenAPIUtils](https://github.com/awslabs/mxnet-model-server/blob/master/frontend/server/src/main/java/com/amazonaws/ml/mms/openapi/OpenApiUtils.java).